lib/PuppeteerSharp.Tests/BrowserContextTests/DefaultBrowserContextTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace PuppeteerSharp.Tests.BrowserContextTests
+{
+    [Collection("PuppeteerLoaderFixture collection")]
+    public class DefaultBrowserContextTests : PuppeteerPageBaseTest
+    {
+        public DefaultBrowserContextTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task PageCookiesShouldWork()
+        {
+            await Page.GoToAsync(TestConstants.EmptyPage);
+
+            await Page.EvaluateExpressionAsync("document.cookie = 'username=John Doe'");
+            var cookie = (await Page.GetCookiesAsync()).FirstOrDefault();
+            Assert.Equal("username", cookie.Name);
+            Assert.Equal("John Doe", cookie.Value);
+            Assert.Equal("localhost", cookie.Domain);
+            Assert.Equal("/", cookie.Path);
+            Assert.Equal(cookie.Expires, -1);
```

Comment:
To keep the style...
```suggestion
            Assert.Equal(-1, cookie.Expires);
```